core/templates/dev/head/pages/topic_editor/topic_editor_navbar_directive.html

```diff
@@ -54,7 +54,7 @@
   </li>
   <li ng-class="{'active': getActiveTabName() === 'main', 'uib-dropdown': getWarningsCount()}" ng-attr-uib-dropdown="<[getWarningsCount()]>" class="icon">
     <a uib-tooltip="Main Editor" tooltip-placement="bottom" ng-click="selectMainTab()">
-      <i class="material-icons">&#xE254;</i> <span style="font-size: 0.8em;">Topic</span>
+      <i class="material-icons">&#xE254;<span style="font-size: 0.9em;">T</span></i>
```

Comment:
I am not entirely sure about this change, as for a new user who is working with topics for the first time, even this would be confusing, as 'S' can correspond to 'Subtopic' or 'Skills'.
I think it would be better to revert this change.

Reply:
I'm also not sure about this -- as @aks681 says, 'S' is ambiguous. Maybe "SUBTOPIC" or "SKILL" in small letters? But that may be hard to read, so we might need to look for a different way of representing this.

@aks681, this might be a question for a designer perhaps?

Reply:
Yeah, we could ask a designer on their thoughts.

Reply:
Reverted back the changes.